### PR TITLE
doc: mention --show-mappings in crushtool manpage

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/19649

### CODE REVIEW

**Comment by dachary** [comment]
--show-mappings shows the mapping for each value, one line per value

**Comment by smithfarm** [comment]
@dachary What is a "value" in this context?

**Comment by smithfarm** [comment]
Hmm, looking at the `crushtool --help` output and comparing with the manpage, it would seem there is more work here than just mentioning `--show-mappings`.

**Comment by smithfarm** [comment]
For instance, the manpage says "The tool has four modes of operation: --compile, --decompile, --build, and --test", but the usage documentation in the source code does not support this at all.

**Comment by dachary** [comment]
> What is a "value" in this context

It is the value of x which goes from --min-x (default to 0) to --max-x (defaults to 1023) unless it is set explicitly with --x

**Comment by smithfarm** [comment]
@dachary How about now?

**Comment by dachary** [comment]
Note: Each Placement Group (PG) has an integer ID which can be obtained +from ``ceph pg dump`` (for example PG 2.2f means pool id 2, PG id 32). The pool id and the PG ID are combined by a function to get a value which is given to CRUSH to map it to OSDs. crushtool does not know about PG or pools and only runs simulations by mapping values in the range [--min-x,--max-x]

File: doc/man/8/crushtool.rst [hunk 1]
```diff
@@ -94,6 +99,16 @@ documented by the **--help-output** option.
    out of **1024** objects ( **4/1024** ) were mapped to **result size
    == 8** devices only.
 
+.. option:: --show-mappings
+
+   Displays the mappings for a range of PG ids (the bounds of which are
+   ``--min-x``, which defaults to 0, and ``--max-x``, which defaults to 
+   1023) or an individual PG id specified by ``--x``. For example::
```
Comment (dachary) [inline doc/man/8/crushtool.rst]: I would just write "Display the mapping of each value. For example:". I don't think it's a good idea to mention PG ids at all in this context.

**Comment by smithfarm** [comment]
@dachary Please go over it once more.

**Comment by smithfarm** [comment]
(added tracker URL)

**Comment by dachary** [comment]
the help output is verified by a test, I suspect that's what causes the failure

**Comment by smithfarm** [comment]
This PR only modifies the manpage, not the help output.

**Comment by smithfarm** [comment]
Jenkins re-test this please

```
99% tests passed, 1 tests failed out of 166

Total Test time (real) = 1397.12 sec

The following tests FAILED:
	149 - osd-fast-mark-down.sh (Failed)
```

**Comment by smithfarm** [comment]
@dachary Tests passed; OK to merge?

**Pull request merged**